src/App/DocumentObserver.h

```diff
@@ -49,9 +49,9 @@ class AppExport DocumentT
     /*! Constructor */
     DocumentT();
     /*! Constructor */
-    DocumentT(Document*);
+    DocumentT(Document*); // explicit bombs

```

Comment:
What happens if it's explicit? A build failure or (hopefully not) a crash at runtime?